tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -28,7 +28,14 @@ public partial class Startup
         /// </summary>
         static Startup()
         {
-            ManagedProfilerDirectory = ResolveManagedProfilerDirectory();
+            var managedProfilerDirectory = ResolveManagedProfilerDirectory();
+            if (managedProfilerDirectory is null)
+            {
+                StartupLogger.Log("Managed profiler directory doesn't exist. Bailing out.", ManagedProfilerDirectory);
+                return;
+            }
+
+            ManagedProfilerDirectory = managedProfilerDirectory;
```

Comment:
nit: the `managedProfilerDirectory` is unnecessary, and tweaked the log

```suggestion
            ManagedProfilerDirectory = ResolveManagedProfilerDirectory();
            if (ManagedProfilerDirectory is null)
            {
                StartupLogger.Log("Managed profiler directory doesn't exist. Automatic instrumentation will be disabled");
                return;
            }
```